packages/addresses/package.json

```
@@ -1,14 +1,30 @@
 {
   "name": "@zetachain/addresses",
-  "version": "0.0.4",
+  "version": "0.0.8",
   "license": "MIT",
   "author": "zetachain",
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
   },
-  "main": "src/index.ts",
+  "main": "./dist/zetachain-addresses.umd.js",
```

```suggestion
  "main": "./dist/zetachain-addresses.umd.cjs",
```

Should it be like that?

---

nop, it's js